HC.Radek/HC.Radek.JS.Ex2/index.html

```diff
@@ -0,0 +1,25 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title></title>
+    <meta charset="UTF-8">
+      <link href="main.css" type="text/css" rel="stylesheet">
+  </head>
+  <body>
+      <div id="mainHolder">
+          <div id="stage1"><button value="start" onclick=" myController.getQuestions();">start</button></div>
+          <div id="stage2" class="hidden">
+              <div id="question">2222</div>
+              <button id="answer1" onclick="myController.checkQuestion(0);"></button>
```

Comment:
czy nie lepiej to budować w pętli :)? troche głupio tak wygląda answer1, answer2 itp :)